docs/content/contribute/styleguide.md

```diff
@@ -0,0 +1,324 @@
+
+#### OSCAL objects and formal names
+
+In formal documentation, OSCAL objects are given formal names in addition to the identifiers or tags provided to represent them in encoded (machine-readable) representations. These formal names help to identify and distinguish the object as defined, and to confirm the subjects of assertions, for example in specification of software requirements. Commonly, for example, they can appear as listings in tables of contents. Formal names in OSCAL use *title case*, for example <q>Assessment Activity Universally Unique Identifier</q> for a data point marked simply as `uuid` in OSCAL data. Sometimes a formal name echoes an encoded (tag) name: <q>Control</q> is the formal name for the object encoded as `control` or tagged (XML syntax) `<control>`.
+
+When using formal names, use them fully spelled out and capitalized, without further formatting. Of course, the context of use should be OSCAL object types (which are defined with such formal names), not something else. In this context, in other words, they are meaningful as proper nouns.
+
+A complete list of formal names for objects defined in OSCAL schemas can be retrieved using the XPath query `//formal-name` (binding unprefixed names to namespace `http://csrc.nist.gov/ns/oscal/1.0`), over the collection of OSCAL metaschemas.
+
+#### Other OSCAL and non-OSCAL terminology
+
+In general, technical terms (and even formal names) should be avoided in favor of non-technical language except when necessary for specific clarification *within the context of use* including your intended audience. For instance, a tutorial intended for beginners does not (and should not) exhaust every topic: it should be technical only as appropriate and necessary, and technical language not actually useful to understand problems or concepts at hand, can often be saved for elsewhere.
+
+Nonetheless, technical terminology is often necessary. For documents reaching non-technical audiences (or non-domain experts), this often requires offering brief definitions for terms at the head of a document, or even a glossary. Even technical audiences can be helped by a link to a specification that defines formal terms. In any case, formal terminologies can be called out and identified as such. For example, both OSCAL and RMF (Risk Management Framework) terminology may combine in a description of a use case for a new feature. In this instance a brief notice of what is meant by terms such as <q>profile</q> or <q>baseline</q> is not out of order.
```

Comment:
Resurfacing the previous comment about acronyms, can we use this to drive our own use of the terminology page, the CSRC glossary, and a hierarchy of more general resources we ask people to rely upon? (And to be fair: I address this to the whole team.)

Reply:
We can include links to glossaries / acronym pages; I have already included a couple.